resources/views/includes/bottomnav.blade.php

```diff
@@ -0,0 +1,19 @@
+      <!-- bottom navigation tab bar -->
+      <div class="navbar">
+          <a href="/alletermine" class="col-4 active">
+              <svg fill="currentColor" viewBox="0 0 20 20">
```

Comment:
SVG code can be saved in svg files and be imported in an img tag for example. Like that it is less likely that somebody will accidently change it ;)